main/dropbox/dropbox_functions.inc.php

```diff
@@ -84,11 +84,11 @@ function handle_multiple_actions()
     if (strstr($_POST['action'], 'move_')) {
         // check move_received_n or move_sent_n command
         if (strstr($_POST['action'], 'received')) {
-              $part = 'received';
-              $to_cat_id = str_replace('move_received_', '', $_POST['action']);
+                $part = 'received';
+                $to_cat_id = str_replace('move_received_', '', $_POST['action']);
```

Comment:
This is not correct either. It replaces 2 spaces with 4 spaces. That's OK, but then it misses the fact that there are now 8 spaces of indentation with the upper level...